src/components/transactions/FlowCommons/TxModalDetails.tsx

```diff
@@ -92,13 +93,18 @@ export const DetailsNumberLineWithSub = ({
   amount,
   amountUSD,
   hideSymbolSuffix,
+  dustRemainingWarning,
```

Comment:
Better give it a text props or sth similar generic as this will open a can of worms.
It's about making the text red, not about dust.

Reply:
Switched it to a text prop called color